Resources/Prototypes/Maps/ovni.yml

```diff
@@ -1,12 +1,12 @@
 - type: gameMap
   id: Ovni
-  mapName: 'Ovni Station'
+  mapName: 'Ovni'
   mapPath: /Maps/ovni.yml
   minPlayers: 0
   maxPlayers: 30
   stations:
     Ovni:
-      mapNameTemplate: '{0} NSS Platillo {1}'
+      mapNameTemplate: 'NSS Platillo "Ovni" {1}'
```

Comment:
What does the `{0}` which was removed here signify?

Reply:
It's part of the random prefix generator. It would give another prefix to the station. This was pointed out by ODJ as redundant and I felt the need to get rid of it.

Reply:
The other stations have {0} though.

Reply:
The point being that the original desire was to have Ovni have the prefix "NSS." In the PRed change, this is the case, but it won't be "NT-1X NSS Platillo "Ovni" 2342" and instead be "NSS Platillo "Ovni" 2342"

Reply:
Alright. So is it Platillo or Ovni? We should probably have a single name for the station instead of the situation where we had Lighthouse/Pharos for a time until Lighthouse became the official name.

Reply:
To be transparent, I asked Andrew Mower about what to call the map and he said "NSS Platillo" so that's what I put there. We could just change it to "{0} Ovni Station {1}"